.maintain/monitoring/healthcheck-lambda.js

```diff
@@ -0,0 +1,185 @@
+const https = require('https');
+const http = require('http');
+const url = require('url');
+const webhookUrl = '{SLACK NOTIFCATION WEBHOOK URL}';
+const port = 9933;
+const apiUrls = [
+  {
+    network: 'Mainnet',
+    url: '{PUT IP/HOSTNAME HERE}',
+    name: 'PoS Validator #1',
+    region: 'us-east-2'
+  },
+  {
+    network: 'Mainnet',
+    url: '{PUT IP/HOSTNAME HERE}',
+    name: 'PoS Full Node',
+    region: 'us-west-1'
+  }
+];
+
+async function parseStatus(status, node) {
```

Comment:
This function does more than parsing status so should be appropriately named

Reply:
refactored and renamed